esrally/driver/driver.py

```diff
@@ -402,13 +402,13 @@ def __init__(self):
 
     @actor.no_retry("task executor")  # pylint: disable=no-value-for-parameter
     def receiveMsg_StartTaskLoop(self, msg, sender):
-        self.parent = sender
+        self.task_preparation_actor = sender
```

Comment:
`self.parent` is still used in various other methods (including in `__init__`)

Reply:
I'm not sure what happened there - I did go through and change there, I did the same with children and then found that was used in the RallyActor class, so reverted that - perhaps I managed to revert some of the parent changes too...will fix